spec/requests/user/settings_spec.rb

```diff
@@ -106,8 +105,8 @@ def upload_logo(file)
           expect(page).to have_field("Upload", visible: false)
         end
         click_on "Update settings"
-        wait_for_ajax
         expect(page).to have_alert(text: "Changes saved!")
+        sleep 0.5 # Since the previous Alerts takes time to disappear, checking avatar_url returns early before the api call is complete
```

Comment:
Since the test has consecutive form submissions that both show "Changes saved!", Capybara's have_alert matcher can match the existing alert before the second API request completes, causing a race condition, so we need to add a wait here in these conditions to let the API complete the request.

Previously, we had full page reloads on submissions, so the previous alerts got dismissed, but now that does not happen

referencing test [failure](https://github.com/antiwork/gumroad/actions/runs/20246240294/job/58127433356?pr=2331)